src/agent/src/netlink.rs

```diff
@@ -398,6 +417,23 @@ impl Handle {
                 .protocol(boot_prot)
                 .scope(scope);
 
+            let message = request.message_mut();
+
+            // calculate the Flag vec from the u32 flags
```

Comment:
Why do we need to parse u32 flag to vec? Not just use something like `RouteFlag::Other()` directly?

Reply:
Because the route.flags here is an int32 value obtained by bit flag bitwise AND, not a value corresponding to an enum type. So here we need to split this bit flags into multiple enum types.